### PR TITLE
inverted order of conditionals in GetUrlLink, and renamed function. F…

### PR DESCRIPTION
**Pull request by fph**
…ixes #50.

### CODE REVIEW

File: bibtexbrowser.php [hunk 1]
```diff
@@ -1194,13 +1194,13 @@ function getBibLink($iconurl=NULL) {
   }
 
   /** returns a "[pdf]" link if relevant. modified to exploit the new method, while keeping backward compatibility */
```
Comment (monperrus) [inline bibtexbrowser.php]: could explain in the strategy in the method API doc?

Comment (fph) [inline bibtexbrowser.php]: I am not the original author of the remark "modified to exploit the new method, while keeping backward compatibility", but I have changed the documentation, I hope this is what you wanted.

**Comment by monperrus** [comment]
Great!
 I propose that if no valid field exists, it returns the empty string.
Would you add a test case for checking this?

**Comment by fph** [comment]
@monperrus Sorry, I completely forgot about finishing this. I have rebased onto the current master and added a few tests.

**Pull request merged**